src/configfile.ts

```diff
@@ -291,6 +314,15 @@ export async function parseConfigFile(parsedJson: unknown):
     }
   }
 
+  // Update trace logDir with label per spec
+  const labelFn = makeUniqueSpecLabelFn(benchmarks);
+  for (const spec of benchmarks) {
+    if (spec.browser.trace !== undefined) {
+      spec.browser.trace.logDir =
+          path.join(spec.browser.trace.logDir, sanitizeFileName(labelFn(spec)));
```

Comment:
This is done to ensure each benchmark has its own directory to dump it's log files into. A sample directory structure might look like:
```
logs/
  bench1/
    log-warmup.json
    log-sample01.json
    log-sample02.json
  bench2/
    ...
```